data/Hrishi5111998

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "Hrishikesh Tule",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Web developer **Digital Nomad**. open source enthusiast",
+  "avatar": "https://github.com/Hrishi5111998.png",
+}
+{
```

Comment:
here before flower brace, you should add link attribute!
And after for every list object you should have both opening and closing flower braces.
Expected to be like this
`"link"=[
     {
      "name": "Twitter",
      "url": "https://twitter.com/@hrishikesh7030",
      "icon": "FaTwitter"
    },
   {
    "name": "Instagram",
      "url": "https://instagram.com/@hrishikesh2412",
      "icon": "FaInstagram"
      },
     {
      "name": "LinkedIn",
      "url": "https://linkedin.com/in/hrishikesh-tule",
      "icon": "FaLinkedIn"
      },
     {
      "name": "Github",
      "url": "https://github.com/Hrish5111998",
      "icon": "FaGithub"
      }
  ]
`